src/main/java/teammates/common/datatransfer/FeedbackRubricQuestionDetails.java

```diff
@@ -364,6 +380,11 @@ public String getNewQuestionSpecificEditFormHtml() {
         this.rubricChoices.add("Agree");
         this.rubricChoices.add("Strongly Agree");
         
+        this.rubricWeights.add(0);
+        this.rubricWeights.add(0);
+        this.rubricWeights.add(0);
+        this.rubricWeights.add(0);
```

Comment:
@damithc, just to check, if the instructor does not assign weights, do we want to give the rubrics choices default weights, or should not display the weights (done in this pr)?